yb-voyager/src/utils/utils.go

```diff
@@ -337,6 +337,37 @@ func ContainsAnySubstringFromSlice(slice []string, s string) bool {
 	return false
 }
 
+func PollForMessageFromOffsetInFile(filePath string, offset int64, message string) error {
```

Comment:
Have this function take a timeout. After timeout exceeds, and it can't find the line , it should error out.